tests/pytest/test_namer.py

```diff
@@ -7,4 +11,26 @@ def test_basic_approval():
 
 def test_received_filename():
     namer = get_default_namer()
-    assert namer.get_received_filename().endswith("ApprovalTests.Python/tests/pytest/test_namer.test_received_filename.received.txt")
+    assert namer.get_received_filename().endswith(
+        os.path.normpath("ApprovalTests.Python/tests/pytest/test_namer.test_received_filename.received.txt"))
+
+
+@pytest.fixture
+def pytest_verify(request):
```

Comment:
I was unsure where to place this fixture, so I placed it here for now. It should probably move before the code is merged.